src/js/components/shepherd-button.svelte

```diff
@@ -3,7 +3,13 @@
   import { isFunction } from '../utils/type-check';
 
   export let config, step;
-  const { action, classes, secondary, text, label } = config;
+
+  $: action = config.action;
```

Comment:
@VladimirAnaniev would it be possible to make `config` itself reactive, rather than each individual property? It feels like there should be a less verbose way to accomplish this.

Reply:
Config itself is reactive, but on line 6 we are getting the initial value and never updating it afterwards

Reply:
I would love to make it more concise but could not find a better solution. The destructuring must be a part of variable declaration and that produced warnings with the label syntax.

Do you have an idea?

Reply:
Nice, glad that works!